### ISSUE TITLE
Cannot open ToDo.txt on USB drive, cannot create ToDo.txt on USB drive

### ISSUE THREAD

**Issue by TriggerDingus** [ransome1/sleek#391]
I did a search and don't find this bug as being reported yet.
I have all of my data on a USB drive so I can easily switch between computers. The problem is when I open Sleek and try to open my ToDo.txt file, there is no way for me to navigate to my USB drive because the "Welcome to Sleek" dialog doesn't see that my USB drive is mounted. For reference, I have attached a screen cap of the Sleek "Open" dialog box, along with one of my file manager where my USB drive is visible.

As a test, I tried creating the file on the USB drive but the result was the same in that Sleek doesn't see the USB drive.

**Desktop (please complete the following information):**
 - OS: Pop!_OS 22.04
 - Version of sleek: 1.1.9
 - Source: Same resule from Pop! Shop and Flathub
 
Sleek Open Dialog:
![SleekCap](https://user-images.githubusercontent.com/75340371/175311900-04266a45-ef96-4783-9f78-c84d2e8bd548.png)

File Manager - USB drive mounted and available but invisible to Sleek
![ActualDrivesAvailableCap](https://user-images.githubusercontent.com/75340371/175311939-c6f3fcc6-5666-400c-b5dd-ca0fa778b08f.png)

**Comment by ransome1** [comment]
@TriggerDingus thank you for reporting this, I'll add it to the backlog. As a workaround you can either install it using Snap (here we recently added support for external discs) or use the AppImage. The latter also updates itself.

**Comment by ransome1** [comment]
@TriggerDingus I just published v1.2.1 on Flathub. Its manifest contains the typical Linux mount points:

```
- --filesystem=home
- --filesystem=/mnt
- --filesystem=/media
- --filesystem=/run/media
```

Could you please download 1.2.1 (sometimes it needs a couple of minutes/hours, before it is availbale) and give me a quick feedback if you can now access your drive?

**Comment by TriggerDingus** [comment]
> @TriggerDingus I just published v1.2.1 on Flathub. Its manifest contains the typical Linux mount points:
> 
> ```
> - --filesystem=home
> - --filesystem=/mnt
> - --filesystem=/media
> - --filesystem=/run/media
> ```
> 
> Could you please download 1.2.1 (sometimes it needs a couple of minutes/hours, before it is availbale) and give me a quick feedback if you can now access your drive?

I checked updates in my Pop_Shop repository and it was already showing up there, so I pulled the download from there. I tested tried creating a new ToDo to my USB drive and also tested opening an existing one on my USB drive - both test cases worked!  Thanks much!

**Comment by ransome1** [comment]
Thanks for your feedback.
